### ISSUE TITLE
Some commands fail with VALUE ERROR: Undefined name: Process

### ISSUE THREAD

**Issue by rikedyp** [aplteam/APLGit2#1]
```
      ]APLGIT2.Version
0.15.1+216
      ]APLGIT2.ListBranches
* Command Execution Failed: VALUE ERROR: Undefined name: Process
```

**Comment by aplteam** [comment]
Cannot reproduce on Windows, Linux, Mac OS.

**Comment by rikedyp** [comment]
I just worked out it's .NET

APLGit2 works with .NET 4.8, but not with .NET 3.1 or 6.0

**Comment by aplteam** [comment]
Fixed with v0.15.2 which does not use .NET at all anymore.
